src/package/Microsoft.TestPlatform.CLI/Microsoft.TestPlatform.CLI.csproj

```diff
@@ -23,6 +23,8 @@
     <NuspecBasePath>$(OutputPath)</NuspecBasePath>
     <PackageId>Microsoft.TestPlatform.CLI</PackageId>
     <PackageTags>vstest visual-studio unittest testplatform mstest microsoft test testing</PackageTags>
+    <PackageProjectUrl>https://github.com/microsoft/vstest</PackageProjectUrl>
```

Comment:
dunno why this is not picking up in source build, all the published nugets have link to repo (when I look at nuget.org). Let's merge now, and figure it out later.